examples/fhir/lab_order.json

```diff
@@ -0,0 +1,118 @@
+{
+  "resourceType": "Bundle",
+  "id": "278637ea-386c-4e24-8aad-d3b2ec088332",
+  "type": "message",
+  "timestamp": "2023-02-23T15:42:22.877-06:00",
+  "entry": [
+    {
+      "resource": {
+        "resourceType": "MessageHeader",
```

Comment:
Good point.  I can add the `source`.

As for the `destination` and `focus`, I have no idea what I would put for them.  Yes, I could make something up.  Since they have an optional cardinality, I'm inclined to not put them in until we have stronger evidence that indicates we need them.

Reply:
Added the `source`.

Reply:
The `destination` we can probably leave off for now or just put a fake lab... it would be the lab that the order is going to.

The `focus` should be the Patient.